### ISSUE TITLE
DS4 Windows interrupted by certain foreground programs

### ISSUE THREAD

**Issue by Anax7** [Jays2Kings/DS4Windows#304]
I use the controller touch pad to move the cursor in windows 10, when certain programs get in the foreground the input from my controller is canceled. It happens all the time with the Task Manager for example. When it gets minimized the cursor control via the touch pad continues to work. This also happened when I tried a game yesterday (Cold Waters) where I want mouse emulation through DS4 Windows. When the game is in foreground the cursor (or any other input) stop responding. I've seen it happening on other System dialogues e.g. OS dialogue to allow UAC. Is there any way to overcome this?

Thanks for reading.

**Comment by nexusdude365** [comment]
I know this is late, but, I've also had this exact issue before. The solution is to run DS4 Windows as an administrator. Right-click on DS4Windows.exe and go to properties>compatibility>run this program as an administrator, then click on apply>ok. There is a drawback to doing this, however, it seems that DS4 will fail to launch on startup. This means you have to manually start DS4 Windows via a hotkey. In my case, I have DS4 set on the third spot in my taskbar and simply click on the Windows Key + 3 to launch it. Hope this helps!

**Comment by nexusdude365** [comment]
Also, I forgot to mention, restart your device after setting it as an administrator for it to take effect.

**Comment by Anax7** [comment]
Brilliant! I had this problem with other games as well, that fixed it! Much appreciated!

**Comment by nexusdude365** [comment]
Glad it worked for you, happy to help!

**Comment by shad00m** [comment]
you can move the mouse WHILE the UAC prompt appears?

**Comment by nexusdude365** [comment]
Yes, this should be the case as what you're doing is giving DS4 administrator privileges. Privileges that should allow it to work during UAC prompts.

**Comment by Ryochan7** [comment]
The mouse will not move while a UAC prompt appears even if DS4Windows is run as the administrator. There are only a few ways around the problem that I know of but none are implemented. List from most feasible to least.

1. Replace SendInput with vmulti
2. Pay around $200 to Microsoft to have DS4Windows as an officially signed application so it can have the ui access flag enabled and enforced
3. Perform a miracle and port DS4Windows to TempleOS

**Comment by shad00m** [comment]
Is creating a service that runs with localsystem privileges also an option?
I think that's how JoyXoff does it

**Comment by hwamil** [comment]
So, this is never going to be solved? That sucks.
Why does Steam Controller have the ability to fallback to "lizard" mode but ds4windows does not?

**Comment by Ryochan7** [comment]
> So, this is never going to be solved? That sucks.

If you really want the problem fixed, give me around $200 so I can properly create signed Windows drivers. I am serious and don't call me Shirley.

![serious_shirley](https://user-images.githubusercontent.com/581968/87054901-46490800-c1c9-11ea-96db-6e2d979a762c.gif)

> Why does Steam Controller have the ability to fallback to "lizard" mode but ds4windows does not?

Hardware events vs. Software events
